code/modules/chemistry/Reagents-Medical.dm

```diff
@@ -436,12 +436,19 @@ datum
 				if(!volume_passed)
 					return
 				volume_passed = clamp(volume_passed, 0, 10)
+				var/max_effectiveness_at = M.max_health*0.25
+				var/min_effectiveness_at = M.max_health*SYNTHFLESH_MINIMUM_HEALTH_PCT
+				var/min_effectiveness = 0.25
+				var/healing_effectiveness = 0
+				var/health_remaining = (M.max_health - M.get_brute_damage() + M.get_burn_damage())
```

Comment:
burn damages counts the wrong way as it is now (same for the other affected chems)